app/lib/common/dialogs/attachment_confirmation.dart

```diff
@@ -1,61 +1,43 @@
 import 'dart:io';
 
 import 'package:acter/common/utils/utils.dart';
-import 'package:acter/common/widgets/default_dialog.dart';
+import 'package:acter/common/widgets/attachments/post_attachment_selection.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart'
+    show AttachmentsManager;
```

Comment:
This is import pattern where we just want to expose certain ffi object. We use it to avoid naming conflicts on flutter side. It's not important but recommended.